rules/rollup.go

```diff
@@ -30,61 +30,38 @@ import (
 	"github.com/m3db/m3metrics/filters"
 	"github.com/m3db/m3metrics/generated/proto/schema"
 	"github.com/m3db/m3metrics/policy"
+	"github.com/m3db/m3metrics/rules/models"
 
 	"github.com/pborman/uuid"
 )
 
 var (
-	emptyRollupTarget RollupTarget
+	emptyRollupTarget rollupTarget
 
 	errRollupRuleSnapshotIndexOutOfRange = errors.New("rollup rule snapshot index out of range")
 	errNilRollupTargetSchema             = errors.New("nil rollup target schema")
 	errNilRollupRuleSnapshotSchema       = errors.New("nil rollup rule snapshot schema")
 	errNilRollupRuleSchema               = errors.New("nil rollup rule schema")
 )
 
-// RollupTargetView is a human friendly representation of a rollup rule target at a given point in time.
-type RollupTargetView struct {
-	Name     string
-	Tags     []string
-	Policies []policy.Policy
-}
-
-func newRollupTargetView(target RollupTarget) RollupTargetView {
-	return RollupTargetView{
-		Name:     string(target.Name),
-		Tags:     stringArrayFromBytesArray(target.Tags),
-		Policies: target.Policies,
-	}
-}
-
-// RollupTarget creates a rollup target from a rollup target view.
-func (rtv RollupTargetView) RollupTarget() RollupTarget {
-	return RollupTarget{
-		Name:     []byte(rtv.Name),
-		Tags:     bytesArrayFromStringArray(rtv.Tags),
-		Policies: rtv.Policies,
-	}
-}
-
-func rollupTargetViewsToTargets(views []RollupTargetView) []RollupTarget {
-	targets := make([]RollupTarget, len(views))
+func rollupTargetViewsToTargets(views []models.RollupTargetView) []rollupTarget {
```

Comment:
nit: I know this is not your change, can we rename this to `newRollupTargetsFromView` and move this to `newRollupTargetFromView`?

Reply:
sure